### DISCUSSION TITLE
v1.11 - Release date

### DISCUSSION THREAD

**Discussion by tznind** [tui-cs/Terminal.Gui#2638] [General]
Appreciate that life comes first but any update on planned release date for 1.11 @tig ? I'd like to get https://github.com/PowerShell/GraphicalTools/pull/179 finished off and the search feature would be a really nice addition to it I think.

**Comment by tig** [comment]
I'll get to it asap!

**Comment by tig** [comment]
https://github.com/gui-cs/Terminal.Gui/releases/tag/v1.11.1

Enjoy!

**Reply by tznind** [reply]
Thanks for doing this @tig but it looks like the Filter is not in the tagged release?  The change log doesn't show Filtering and the changes don't seem to be shipped when I draw down the nuget package:

Files are in `develop`
https://github.com/gui-cs/gui.cs/blob/develop/Terminal.Gui/Views/ITreeViewFilter.cs

And in `main`
https://github.com/gui-cs/gui.cs/blob/main/Terminal.Gui/Views/ITreeViewFilter.cs

But not in `1.11.1`
https://github.com/gui-cs/Terminal.Gui/blob/v1.11.1/Terminal.Gui/Views/ITreeViewFilter.cs

`1.11.1` shows "5 commits since this release" and those include the filtering
https://github.com/gui-cs/Terminal.Gui/releases/tag/v1.11.1

**Comment by tig** [comment]
No idea what I did to do that. Fixed with 1.11.2 now, I hope!
